pages/about/groups/task-forces/wcag2ict/work-statement.md

```diff
@@ -42,15 +42,17 @@ The work of the task force includes:
 *   Producing an updated Working Group Note (a form of W3C Technical Report)—following the W3C process under the coordination, review, and approval of the AG WG—that describes:
     *   how each of the new WCAG 2.x A, AA, and AAA success criteria could apply to non-Web ICT;
     *   how each of the new WCAG 2.x guidelines could apply to non-Web ICT;
-    *   clarification of challenges of applying particular WCAG 2.x success criteria to non-web ICT, including closed product software; and
-    *   address challenges in [Open issues labeled WCAG2ICT](https://github.com/w3c/wcag2ict/issues?q=is%3Aissue+is%3Aopen+label%3Awcag2ict).
+    *   clarification of challenges of applying particular WCAG 2.x success criteria to non-web ICT, including closed product software;
+    *   address challenges in [Open issues labeled WCAG2ICT](https://github.com/w3c/wcag2ict/issues?q=is%3Aissue+is%3Aopen+label%3Awcag2ict); and
+    *   provide an explainer that clarifies the usage of WCAG2ICT
 
 Explicitly out of scope for the WCAG2ICT Task Force work are:
 
 *   proposing changes to WCAG 2.x;
 *   developing techniques or interpretations of WCAG 2.x for implementing WCAG 2.x in Web technologies;
 *   developing any specific, sufficient techniques, including platform-specific techniques, for implementing WCAG 2.x in non-Web ICT, though examples may be in scope;
-*   determining whether WCAG, as a whole, _should be_ applied to non-web content; and
+*   determining whether WCAG, as a whole, _should be_ applied to non-web content;
+*   providing a determination of which success criteria are applicable to a particular non-web technology; and
```

Comment:
I am of the opinion that this line should be moved to in scope!  (But maybe soften "determination" to recommendation.

Reply:
@bruce-usab Per the 18 November discussion with the AG WG leadership on changing the scoping and the TF discussion of changes on 21 November, I've made adjustments to the in scope and out of scope bullets.